packages/iris-grid/src/IrisGridTableModel.ts

```diff
@@ -16,6 +16,8 @@ import { PartitionedGridModelProvider } from './PartitionedGridModel';
 
 const log = Log.module('IrisGridTableModel');
 
+const EMPTY_ARRAY: readonly string[] = [];
```

Comment:
You can import `EMPTY_ARRAY` from `@deephaven/utils` and use that, we don't need to redefine it.